Automattic/wp-calypso
# Launchpad: Added skip button to fullscreen Launchpad

More context: paYKcK-3m0-p2

Related to https://github.com/Automattic/jetpack/pull/32500
Related to https://github.com/Automattic/wp-calypso/issues/76876
~~Related to D120019-code~~

## Proposed Changes

- This PR makes the `Skip for now` button on the top right corner persist the `skipped` option, so users can access the customer home. 
- After skipping this step, users won't be forced to the fullscreen Launchpad anymore.
- Until the site is launched, the pre-launch checklist is going to be shown in the customer-home page.
- `<LaunchpadPreLaunch />` is a new component that renders pre-launch checklist in customer-home.

**This PR covers the write intent only.**

## Testing Instructions



- Access calypso.live
- Create a site with `/start`, use the `write` intent
![image](https://github.com/Automattic/wp-calypso/assets/3801502/78023d72-a145-4d6a-80f6-ee90d2860bb1)

- You will be redirected to fullscreen Launchpad
- Click on the `Skip for now` button
![image](https://github.com/Automattic/wp-calypso/assets/3801502/8fb83d66-a4d0-4301-830c-e04a25c23324)

- You should see the same checklist
![image](https://github.com/Automattic/wp-calypso/assets/3801502/9c26ab33-d568-47ec-9169-a9c767cd33e5)

- Click on the Launch your site task
- Check the celebrate modal is shown
 
![image](https://github.com/Automattic/wp-calypso/assets/3801502/8578b0c1-6c42-4d1d-b486-1315d2f63301)

- You should now see the post-launch checklist

![image](https://github.com/Automattic/wp-calypso/assets/3801502/a780cdf3-abbe-447a-ab8b-e4b3d394d7bb)
